known2/php7/0.9.9/init.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0"?>
+<installer>
+
+	<information>
+		<info id="version" value="0.9.9"/>
+		<info id="license" value="_apps_opensource"/>
+		<info id="date" value="2019-01-22"/>
+		<info id="cost" value=""/>
+	</information>
+
+	<links>
+	</links>
+
+        <requirements>
+		<requirement id="itron-version" value="5.0.0"/>
+		<requirement id="diskspace" value="50"/>
+		<requirement id="database" value="1"/>
+		<requirement id="db-type" value="mysql"/>
+		<requirement id="db-prefix-support" value="0"/>
+		<requirement id="php-url-fopen" value="1"/>
+		<requirement id="php-register-globals" value="0"/>
+		<requirement id="php-magic-quotes" value="0"/>
+		<requirement id="php-safe-mode" value="0"/>
+		<requirement id="php-open-basedir" value="0"/>
+		<requirement id="php-short-open-tag" value="1"/>
+		<requirement id="php-version-minimum" value="5.4"/>
+	</requirements>
+
+	<skeleton>
+		<file id=".gitignore">
+		<file id=".htaccess">
+		<file id=".travis.yml">
+		<file id="ansible.sample">
+		<file id="chrome">
+		<file id="config.ini" isconfig="true"/>
+		<file id="ConsolePlugins">
+		<file id="CONTRIBUTORS.md">
+		<file id="css">
+		<file id="docs">
+		<file id="docs-mkdocs.yml">
+		<file id="doxygen.conf">
+		<file id="external">
+		<file id="gfx">
+		<file id="HARASSMENT-POLICY.md">
+		<file id="htaccess-2.4.dist">
+		<file id="htaccess.dist">
+		<file id="Idno">
+		<file id="IdnoPlugins">
+		<file id="index.php">
+		<file id="ISSUE_TEMPLATE">
+		<file id="js">
+		<file id="known.php">
+		<file id="LICENSE">
+		<file id="mkdocs.yml">
+		<file id="nginx.conf">
+		<file id="phpunit.xml">
+		<file id="PULL_REQUEST_TEMPLATE">
+		<file id="README.md">
+		<file id="schemas">
+		<file id="statics">
+		<file id="templates">
+		<file id="Tests">
+		<file id="Themes">
+		<file id="Uploads">
+		<file id="Vagrantfile.sample">
+		<file id="version.known">
+		<file id="warmup">
+	</skeleton>
+
+	<archives>
+		<archive id="main" url="http://assets.withknown.com/releases/known-0.9.9.tgz" type="tar.gz" md5="728d8dc6bf4a8c7dd9cdb2ca58d2fe6a"/>

```

Comment:
Nice, thanks for looking into this! 

Suggestion, I'd definitely consider using the 1.0.0 build at least - 0.9.9 is basically prehistoric at this point.

Reply:
Did you know it was collapsed in the releases channel (never visited the repo before, so perhaps something that trips up others)